src/rotation_generator.jl

```diff
@@ -246,3 +246,30 @@ function Base.show(io::IO, ::MIME"text/plain", X::RotationGenerator)
     io = IOContext(io, :typeinfo => eltype(X))
     Base.print_array(io, X)
 end
+
+"""
+    isrotationgenerator(r)
+    isrotationgenerator(r, tol)
```

Comment:
Do we want a nonzero tolerance by default?  Something like a small multiple of `eps(eltype(r))`?

Reply:
Ah, I just copied the docstring from `isrotation`. I'm planning to remove the `tol` argument.